tests/pkg_tflite-micro/Makefile

```diff
@@ -0,0 +1,14 @@
+# Ensure minimal size by default
+DEVELHELP ?= 0
+
+include ../Makefile.tests_common
+
+USEPKG += tflite-micro
+
+# TensorFlow-Lite crashes on M4/M7 CPUs when FPU is enabled, so disable it by
+# default for now
```

Comment:
Can you open an issue for that? 
Smells fishy.

Reply:
> Can you open an issue for that?

Sure. I tested without just to be sure and on cortexm it ends up with a hardfault. Maybe that's the same issue on riscv ?

> Does this package make use of the DSP instructions found in those cores? Might be that we are not handling those registers properly yet.

There's room for improvements. On cortex m4/7, we could enable CMSIS-NN primitives to speed up the inference (40% gain).
More generally, we could package kissfft and see how to support more complex scenario for keyword spotting. I was not able to make the whole stuff to work when I ported tensorflow-lite (back in 2019/2020) but maybe things have evolved positively since then.
I'm also looking into the [edge-impulse SDK](https://github.com/edgeimpulse/inferencing-sdk-cpp) (I have a [branch](https://github.com/aabadie/riot/tree/pr/pkg/edge-impulse-sdk)) which also relies on tensorflow and is easier to package, all third-party libraries are present in the repo, so can adapted with only one package.